corehq/apps/es/client.py

```diff
@@ -628,7 +633,15 @@ def _search(self, query, **kw):
         """Perform a "low-level" search and return the raw result. This is
         split into a separate method for ease of testing the result format.
         """
-        return self._es.search(self.index_name, self.type, query, **kw)
+        with metrics_histogram_timer(
+                'commcare.elasticsearch.search.timing',
+                timing_buckets=(1, 10),
```

Comment:
This splits results into 3 buckets - again, I'm open to suggestions here, just not sure we care much beyond this.  I figure less than a second is generally fast enough, between 1 and 10 seconds is slow, and over 10 seconds is concerning.  Note that we do also have access to the overall average query duration already (just not broken down by domain).

Reply:
Those three buckets seem good to me.